db/subscriptions.go

```diff
@@ -80,6 +80,38 @@ func RemoveFromWatchlist(userId uint64, validator_publickey string, network stri
 	return err
 }
 
+func RemoveFromWatchlistBatch(userId uint64, validator_publickeys []string, network string) error {
+	keys := [][]byte{}
+	for _, keyString := range validator_publickeys {
+		key, err := hex.DecodeString(keyString)
+		if err != nil {
+			return err
+		}
+		keys = append(keys, key)
+	}
+	tx, err := FrontendWriterDB.Begin()
+	if err != nil {
+		return fmt.Errorf("error starting db transactions: %v", err)
```

Comment:
You have to use %w, otherwise the error isn't correctly nested.

Reply:
wow first time I learn that there is a %w after half a year ;-) 
fixed